lib/salus/scanners/base.rb

```diff
@@ -53,6 +53,12 @@ def self.supported_languages
       []
     end
 
+    # Must be overriden by report scanners and should return true.
+    # This will be used to unblock other scanners which are dependent on Reporting Scanners output
+    def is_reporting_scanner?
```

Comment:
If we find additional reporting feature needs we could have our scanner subclass a common report scanner class and then simply use .kind_of?(ReportScanner) here and remove the need for each class to define a is_reporting_scanner?

Reply:
I looked at a bunch of `Report...` scanners, and all have the function defined below, and it doesn't look like `SBOM_REPORT` is used in any non-`Report...` scanners.  Perhaps just use that info to determine if the scanner is a reporting scanner?
Though you may want to double check if each of the report scanners has `SBOM_REPORT` as its type.

```
    def self.scanner_type
      Salus::ScannerTypes::SBOM_REPORT
    end
```

Reply:
@joshuaostrom Scanner_type also seems to be a good way of knowing if its a reporting scanner, I can pivot to using this as well

Reply:
That works for me!! Thanks all!

Reply:
Done